### PR TITLE
Update enable-tab-completion.rst

### PR DESCRIPTION
**Pull request by jay-tau**
### Description
To deal with https://github.com/conda/conda/issues/11171

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [x] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

**Comment by jay-tau** [comment]
Hello,
I was wondering if anything needs to be done before this PR can be merged.
Cheers,
jaytau

**Review by beeankha — DISMISSED** [review]

File: news/11738-update-tab-completion-docs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+* <news item>
+
+### Deprecations
+
+* <news item>
+
+### Docs
+
+* Optional Bash completion support has been removed starting in v4.4.0, and not just deprecated (https://github.com/conda/conda/issues/11171)
```
Comment (beeankha) [inline news/11738-update-tab-completion-docs:15]:
```suggestion
* Optional Bash completion support has been removed starting in v4.4.0, and not just deprecated (#11171)
```

Comment (beeankha) [inline news/11738-update-tab-completion-docs:15]:
```suggestion
* Optional Bash completion support has been removed starting in v4.4.0, and not just deprecated. (#11171)
```

**Review by beeankha — APPROVED** [review]
Thanks for making this change, @jay-tau !

**Pull request merged**